lib/src/ast/element.dart

```diff
@@ -45,11 +51,11 @@ abstract class ElementAst implements StandaloneTemplateAst {
   /// Create a new element AST from parsed source.
   factory ElementAst.parsed(
     SourceFile sourceFile,
-    NgToken beginToken,
+    NgToken openElementStart,
     NgToken nameToken,
-    int openTagEnd,
-    int closeTagStart,
-    NgToken endToken, {
+    NgToken openElementEnd, {
+    bool isVoidElement,
```

Comment:
Oh this was left as a barrier for potential incomplete voidElement list in parser.dart - but will be removed shortly.